packages/react-accordion/src/components/AccordionPanel/useAccordionPanelStyles.ts

```diff
@@ -0,0 +1,23 @@
+import { makeStyles, ax } from '@fluentui/react-make-styles';
+import { AccordionPanelState } from './AccordionPanel.types';
+
+/**
+ * Styles for the root slot
+ */
+const useRootStyles = makeStyles<AccordionPanelState>([
+  [
+    null,
+    {
+      marginInline: '12px',
```

Comment:
According to MDN this does not work even in latest Edge without flags, can we use a different approach for styling there?

https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline#browser_compatibility